docs/document/content/user-manual/shardingsphere-proxy/startup/bin.cn.md

```diff
@@ -45,6 +45,9 @@ weight = 1
 当用户需要使用自定义的算法类时，可通过以下方式配置使用自定义算法，以分片为例：
 
 1. 实现 `ShardingAlgorithm` 接口定义的算法实现类。
+1. 在项目 resources 目录下创建 META-INF/services 目录。
+1. 在 META-INF/services 目录下新建文件 org.apache.shardingsphere.sharding.spi.ShardingAlgorithm
```

Comment:
META-INF/services and org.apache.shardingsphere.sharding.spi.ShardingAlgorithm should around `